sound/soc/sof/intel/hda-dsp.c

```diff
@@ -453,6 +453,15 @@ static int hda_resume(struct snd_sof_dev *sdev, bool runtime_resume)
 
 int hda_dsp_resume(struct snd_sof_dev *sdev)
 {
+	struct pci_dev *pci = to_pci_dev(sdev->dev);
+
+	if (sdev->s0_suspend) {
+		/* resuming from S0ix. */
+		pci_restore_state(pci);
+		disable_irq_wake(pci->irq);
+		return 0;
```

Comment:
the commit message talks about D0i3 but you meant S0ix here. There is nothing related to D0ix in this patch.

Reply:
Correct, let me change it.